Misc/Positions_Area Positions 4 External.txt

```diff
@@ -103,22 +87,6 @@ EISN_LN_CTR:Shannon Control:119.075:SNLN:A:EISN:CTR:-:-:6600:6677
 
 EISN_I_CTR:Shannon Information:127.500:SNI:I:EISN:CTR:-:-:6600:6677
 
-;Shannon - Withdrawn 2023/11
-;EISN_CTR:Shannon Control:131.150:SNE:E:EISN:CTR:-:-:6600:6677
-
-EISN_B_CTR:Shannon Control:134.275:SNB:B:EISN:CTR:-:-:6600:6667
-EISN_N_CTR:Shannon Control:122.975:SNN:N:EISN:CTR:-:-:6600:6667
-EISN_O_CTR:Shannon Control:129.675:SNO:N:EISN:CTR:-:-:6600:6667
-EISN_S_CTR:Shannon Control:135.600:SNS:S:EISN:CTR:-:-:6600:6667
-EISN_W_CTR:Shannon Control:120.050:SNW:W:EISN:CTR:-:-:6600:6667
-EISN_C_CTR:Shannon Control:129.675:SNC:C:EISN:CTR:-:-:6600:6667
```

Comment:
Ahh yeah, so this line defining `SNC` does need keeping then, guessing it was just added back into the wrong section in all likelyhood. Pop it into the eqvialent place under SNW's definition and should be good hopefully.

Reply:
Added back EISN_C_CTR, validation was successful!